### PR TITLE
fix(Pipelines): Fix multiple float parameters display

### PR DESCRIPTION
**Pull request by cheikhgwane**
Fix display of float parameters (multiple) on run page.

## Screenshots / screencast

image

### CODE REVIEW

File: src/pages/workspaces/[workspaceSlug]/pipelines/[pipelineCode]/runs/[runId].tsx [hunk 1]
```diff
@@ -89,12 +89,9 @@ const WorkspacePipelineRunPage: NextPageWithLayout = (props: Props) => {
     if (entry.type === "bool") {
       return <Switch checked={entry.value} disabled />;
     }
-    if (entry.type === "int" && entry.value !== null) {
+    if ((entry.type === "int" || "float") && entry.value !== null) {
```
Comment (pvanliefland) [inline src/pages/workspaces/[workspaceSlug]/pipelines/[pipelineCode]/runs/[runId].tsx:92]: This doesn't feel right. 

```javascript
if ("yolo" === "int" || "float") {
    console.log("What?");
}
```

**Review by pvanliefland — CHANGES_REQUESTED** [review]
@cheikhgwane I think there is a bug in your code, and the tests don't catch it.

File: src/pages/workspaces/[workspaceSlug]/pipelines/[pipelineCode]/runs/[runId].tsx (hunk 1 above)
Comment (cheikhgwane) [reply src/pages/workspaces/[workspaceSlug]/pipelines/[pipelineCode]/runs/[runId].tsx:92]: Ah I see!

**Review by pvanliefland — APPROVED** [review]
LGTM!

**Pull request merged**